src/components/Layout/Header/Navigation.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NavLink } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronCircleUp, faChevronCircleDown } from '@fortawesome/free-solid-svg-icons'
+
+import LinkWithPastLocation from 'components/LinkWithPastLocation'
+
+export type NavLinkKeys = 'Trade' | 'Wallet' | 'Orders'
+
+export const NavLinksWrapper = styled.div<{ $open?: boolean; $responsive: boolean }>`
+  display: flex;
+  justify-content: space-evenly;
+
+  margin: 0.5rem;
+  padding: 0.5rem;
+
+  white-space: nowrap;
+
+  a {
+    color: var(--color-text-secondary);
+    font-size: 3.2rem;
+    font-weight: 1000;
+    padding: 0.35em;
+    text-decoration: none;
+
+    transition: color 0.2s ease-in-out;
+
+    &:hover {
+      color: var(--color-text-primary);
+    }
+
+    &.active {
+      color: var(--color-text-primary);
+    }
+  }
+
+  ${({ $responsive, $open }): string | false =>
+    $responsive &&
```

Comment:
Any special reason for prefixing the parameters with `$`?

Reply:
yeah as its styled and they are extending normal HTML comps, i dont want to clash there prop name wise